.github/workflows/test.yml

```diff
@@ -56,3 +56,22 @@ jobs:
         run: |
           pip install codecov
           codecov
+
+  lint:
+    runs-on: ubuntu-latest
+    if: always()
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
Can probably drop this? We only need black for this check. Or is this anticipating further linting that needs to import the package?

Reply:
I think you're right, this isn't necessary.
```suggestion
```